config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+
+  root 'users#feed'
+
+  match "/auth/:provider/callback" => "sessions#create", via: [:get, :post]
+  get "/login" => "sessions#new", as: :login
+  delete "/logout/"              => "sessions#destroy", as: :logout
+  get "/search/"                => "creators#search"
```

Comment:
I would not try to line up these routes like this. It's hard to maintain, and doesn't add much value.